AKS/AKSClusterCheck.psm1

```diff
@@ -41,6 +41,17 @@ class AKSClusterCheck: ResourceCheck {
         return $totalNodeCount
     }
 
+    [bool] IsPoolMin3Nodes() {
```

Comment:
For code clarity, please follow the same convention when it comes to case and naming. I would propose something like `hasMin3NodesPerPool`